### PR TITLE
[52] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by gracioust**
## Task: `Develop a Contact Form Page using HTML`
- `Developed
Capturec
 a Contact Form Page using HTML'

### CODE REVIEW

File: submissions/tosinIshola/html-forms/contact_form.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en"><head>
```
Comment (ainobs) [inline submissions/tosinIshola/html-forms/contact_form.html:2]: hello tosin, nice job so far. Please move this head tag down into the next line.

**Review by ainobs — CHANGES_REQUESTED** [review]

File: submissions/tosinIshola/HTML/Ecommerce.html [hunk 2]
```diff
@@ -30,7 +31,9 @@ <h2>ABOUT US</h2>
 
 <h2>OUR PRODUCTS</h2>
 
+
```
Comment (TundeOladejo) [inline submissions/tosinIshola/HTML/Ecommerce.html:34]: Remove this extra line

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/tosinIshola/HTML/Ecommerce.html (hunk 2 above)
Comment (gracioust) [reply submissions/tosinIshola/HTML/Ecommerce.html:34]: What extra line?

File: submissions/tosinIshola/html-forms/contact_form.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <li><a href="">ABOUT US</a></li>
+        <li><a href="">OUR PRODUCT</a> </li>
+        <li><a href="" > CUSTOMER SERVICE</a> </li>
+    </ul>
+
+</nav>
+
+<h1>Contact Form</h1>
+
+<form action="#" method="post" enctype="multipart/form-data">
```
Comment (adaugochi) [inline submissions/tosinIshola/html-forms/contact_form.html:21]: All this `action="#" method="post" enctype="multipart/form-data"` aren't needed. Can you remove them?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**